docs/core/integration-quality-scale/rules/docs-supported-devices.md

```diff
@@ -1,16 +1,17 @@
 ---
-title: "The integration documents known supported / unsupported devices"
+title: "The documentation lists known supported / unsupported devices"
 ---
 
 ## Reasoning
 
 A lot of Home Assistant users buy devices based on if Home Assistant supports them.
-To make it easier for users to find out if a device is supported, the documentation should document the known supported or unsupported devices.
+To make it easier for users to find out if a device is supported, the documentation should list the known supported or unsupported devices.
```

Comment:
I explicitly requested us not to use the word "list" when writing the rules! I don't think it should be a requirement to list the devices. It should be up to each integration how to describe the situation since different situations requires different descriptions.

Reply:
Oh, I missed that. I agree with you. I changed it here only to align with the title, not to emphasize a particular format. which should then be rephrased.

Reply:
https://github.com/home-assistant/developers.home-assistant/pull/2478